### PR TITLE
Bugfix

### PR DESCRIPTION
**Pull request by tomscytale**
Remove some buggy code - it calculated some variables that were never used.
Update README.md - `current_day` should be a string

### CODE REVIEW

File: src/lib-components/kalendar-weekview.vue [hunk 1]
```diff
@@ -129,12 +129,6 @@ export default {
         return null;
       }
 
-      let label = time.split("T")[1].slice(0, 5);
-      if (this.kalendar_options.military_time) {
-        let ampm = formatAMPM(label.split(":")[0]);
-        let ampmlabel = ampm.slice(-2);
-        console.log({ ampm });
-      }
```
Comment (tomscytale) [inline src/lib-components/kalendar-weekview.vue]: this passes a string to `formatAMPM` but it expects a date object and so raises an error. 

This code assigns values to the variables `ampm` and `ampmlabel` but these are never used so it seems that the whole block does nothing.

Comment (altinselimi) [reply src/lib-components/kalendar-weekview.vue]: I see. It seems it was a left over from the rewrite.

**Comment by altinselimi** [comment]
@tomscytale can you do a rebase with master as it seems this PR has a conflicting file.

**Comment by tomscytale** [comment]
done

**Pull request merged**